libr/core/cmd_search_rop.c

```diff
@@ -52,9 +52,101 @@ static RList* get_constants (const char *str) {
 	return list;
 }
 
+static bool isFlag (const char *f) {
+	const char *flags = "zf,pf,sf,cf,of";
+
+	if (strstr (flags, f)) {
```

Comment:
wat this is not portable and wrong way to check if its a flag. you can take this info from rreg